sqs.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_sqs_queue" "keycloak_to_alerts_concierge_user_updates" {
+  name = "keycloak-${var.environment}-alerts-concierge-user-updates"
+  sqs_managed_sse_enabled = true
```

Comment:
Is SQS-managed encryption appropriate, or would we want to use KMS?

Reply:
It looks like SSE-SQS comes with [some caveats](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-server-side-encryption.html), but I don't know if any of them apply to us. Conversely, KMS incurs some small usage costs.

If we use KMS, we should be able to use an AWS-managed key, which they _should_ create for us automatically and name `alias/aws/sqs`. That said, I'm not sure whether we can simply specify `kms_master_key_id = "alias/aws/sqs"` in Terraform and it will "just work".

We can start with SSE-SQS for now and hopefully that will be sufficient?